src/quo/components/profile/link_card/properties.cljs

```diff
@@ -0,0 +1,10 @@
+(ns quo.components.profile.link-card.properties
+  (:require [quo.foundations.colors :as colors]))
+
+(defn gradient-start-color
+  [theme customization-color]
+  (colors/resolve-color customization-color theme 0))
+
+(defn gradient-end-color
+  [theme customization-color]
+  (colors/resolve-color customization-color theme 6))
```

Comment:
Are these worth putting in a separate `properties` file? Maybe can be put in the `view` file or the `style` file as they are calculating a style

Reply:
@OmarBasem It was suggested by the team in another PR to separate them and put it in `properties` files.